### DISCUSSION TITLE
How to integrate graphql with Elasticsearch so that schema gets created using mapping of es index.

### DISCUSSION THREAD

**Discussion by VishnuGupta16** [graphql-java-kickstart/graphql-spring-boot#565] [Q&A]
Hi All,

Need to integrate Elasticsearch via Graphql query so that schema gets created and reloaded at runtime via index mapping.

Something like: https://graphql-compose.github.io/docs/plugins/plugin-elasticsearch.html

**Comment by jatin-abrol** [comment]
hi @VishnuGupta16 I know it has been a while but were you able to get something on this? I am trying to build something similar but having a hard time finding anything relevant
